jsx/FilterableDataTable.js

```diff
@@ -54,6 +54,7 @@ class FilterableDataTable extends Component {
           columns={this.props.columns}
           filter={this.state.filter}
           fields={this.props.fields}
+          actions={this.props.actions}
```

Comment:
i don't understand this. the actions will be rendered in both the filter and the data table?

Reply:
refer to previous comment.